### DISCUSSION TITLE
Request feature to filter with "Group By"

### DISCUSSION THREAD

**Discussion by BerkantCiftci** [directus/directus#10887] [Feature Requests]
In my situation I use the following two tables:

 - Table in which I register my batteries with a unique ID
 - Table in which my batteries send data and link it with their unique ID.

I would like to add a field to the 1st table that shows what my most recent data is. this is in the form of an UPDATE and not a POST!

To make it more clear, I put an example down below:

Click on the *orders* table at the left side on the page. Then add the SQL Query. [w3schools example](https://www.w3schools.com/sql/trysql.asp?filename=trysql_asc).

```SQL 
SELECT EmployeeID, MAX(OrderDate) as OrderDate FROM [Orders]
GROUP BY EmployeeID
```

**Comment by rijkvanzanten** [comment]
There's no way to filter based on unrelated table data at this point (besides fetching the IDs first, and manually adding them in an `_in` clause

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
